articles/quickstart/native/android/interactive.md

```diff
@@ -85,7 +85,7 @@ Run **Sync Project with Gradle Files** inside Android Studio or execute `./gradl
 For more information about using Gradle, check the [Gradle official documentation](https://gradle.org/getting-started-android-build/).
 :::
 
-## Add login to your app {{{ data-action=code data-code="MainActivity.kt#6:38" }}}
+## Add Login to your application {{{ data-action=code data-code="MainActivity.kt#6:38" }}}
 
```

Comment:
^ This is a secondary heading, which is why it has ## so it does not need to be capitalized. "Add login to your application."